didcomm/protocols/forward/forward.py

```diff
@@ -24,11 +60,32 @@ class ForwardResult:
     forwarded_msg_encrypted_to: Optional[List[DID_URL]] = None
 
 
+# TODO move to a helper module
+async def find_did_service(
+    to_did_or_kid: DID_OR_DID_URL, resolvers_config: ResolversConfig
+) -> DIDCommService:
+
+    to_did = get_did(to_did_or_kid)
+    did_doc = await resolvers_config.did_resolver.resolve(to_did)
+
+    if did_doc is None:
+        raise DIDDocNotResolvedError()
+
+    # TODO do not restrict to the first only
+    try:
+        return did_doc.didcomm_services[0]
+    except IndexError:
+        raise DIDServiceNotFoundError
```

Comment:
I think it's totally fine if there are no services, so I don't think that we should have and raise DIDServiceNotFoundError

Reply:
done